inductiva/molecules/scenarios/_post_processing.py

```diff
@@ -0,0 +1,66 @@
+"""Post process Gromacs simulation outputs."""
+from typing import Optional
+
+from absl import logging
+
+import os
+import MDAnalysis as mda
+import nglview as nv
+
+from uuid import UUID
+
+from inductiva.types import Path
+from inductiva.molecules.simulators import GROMACS
+from inductiva.utils.templates import (TEMPLATES_PATH,
+                                       replace_params_in_template)
+from inductiva.tasks import fetch_task_output, get_task_info
+
+import json
+
+SCENARIO_TEMPLATE_DIR = os.path.join(TEMPLATES_PATH, "protein_visualization")
+GROMACS_TEMPLATE_INPUT_DIR = "gromacs"
+
+
+class GROMACSSimulationOutput:
+    """Post process GROMACS simulation outputs."""
+
+    def __init__(self, sim_output_path: Path = None):
+        """Initializes a `SimulationOutput` object.
+        
+        Given a simulation output directory that contains the standard files
+        generated by a GROMACS simulation run, this class provides methods to
+        visualize the simulation outputs in a notebook.
+        
+        Args:
+            sim_output_path: Path to the simulation output directory."""
+
+        self.sim_output_dir = sim_output_path
+
+    def render(self, pdb_file=None, trajectory_name: str = "trajectory.xtc"):
```

Comment:
I don't think it is a good idea to initialize the name of the trajectory file, since it may be error inducing. But maybe mention that we only support xtc types of files!

Reply:
Wdym with "initialize the name of the trajectory file"?